.github/actions/verify-token/dist/index.js

```diff
@@ -50997,14 +51015,28 @@ function getTriggerRef(rawTokenObj) {
     return rawTokenObj.github.ref;
 }
 exports.getTriggerRef = getTriggerRef;
+function validateSha1(sha1) {
```

Comment:
not a real issue I think but strange that this shows up twice in the compiled dist file.

Reply:
I re-generated but still appearing multiple times.